File: src/main/java/br/com/newObjects/model/entities/classes/usuario/NovoCliente.java
package br.com.newObjects.model.entities.classes.usuario;

import br.com.model.entities.classes.usuario.Cliente;
import br.com.newObjects.model.entities.classes.endereco.NovoEndereco;
import br.com.newObjects.model.entities.classes.telefone.NovoTelefone;

public class NovoCliente {

    public static Cliente novoCliente() {
        Cliente cliente = new Cliente("NomeClienteTeste", "000.000.000-00", "teste123456", "teste@gmailCliente.com");
        NovoTelefone.novaListaTelefoneCliente(cliente);
        NovoEndereco.novaListaEnderecoCliente(cliente);

        return cliente;
    }
}

File: src/test/java/br/com/newObjects/model/usuario/NovoCliente.kt
package br.com.newObjects.model.usuario

import br.com.model.usuario.Cliente
import br.com.newObjects.model.NovoEndereco.novaListaEndereco
import br.com.newObjects.model.NovoTelefone.novaListaTelefones

object NovoCliente {

    private val usuario = NovoUsuario.novoUsuario()

    fun novoCliente(): Cliente {
        val cliente = Cliente(
            usuario.nome + "Cliente!", usuario.cnpjCpf, usuario.senha, usuario.email
        )
        novaListaTelefones(cliente)
        novaListaEndereco(cliente)
        return cliente
    }
}
